operator/src/test/java/oracle/kubernetes/operator/helpers/AdminPodHelperTest.java

```diff
@@ -328,6 +330,75 @@ public void whenPodHasDuplicateVolumeMounts_createAdminPodWithCombination() {
             hasVolumeMount("volume1", "/domain-path1"), hasVolumeMount("volume2", "/server-path")));
   }
 
+  @Test
```

Comment:
maybe add a test for the 'I did not add labels or annotations' cases, too.

Reply:
We set internal labels and annotations so its never empty.